src/main/java/io/github/hydos/proton/Proton.java

```diff
@@ -46,7 +46,7 @@ public void onInitialize() {
         }
     }
 
-    public static Identifier identifier(String name) {
-        return new Identifier(MOD_ID, name);
+    public static Identifier identifier(String path) {
```

Comment:
Path?

Reply:
Yes

Reply:
That's the name in the `Identifier` constructor

Reply:
I can revert it

Reply:
Revert it.